react/features/speaker-stats/reducer.ts

```diff
@@ -53,6 +96,44 @@ ReducerRegistry.register<ISpeakerStatsState>('features/speaker-stats',
             showFaceExpressions: !state.showFaceExpressions
         };
     }
+    case ADD_TO_OFFSET: {
+        return {
+            ...state,
+            offsetLeft: state.offsetLeft + action.value,
+            offsetRight: state.offsetRight + action.value
+        };
+    }
+    case SET_OFFSET: {
```

Comment:
think this action is not used. as the corresponding action is not used and no middleware process it.